config/jobs/periodic/kubernetes/perf-tests/perf-tests-kubernetes-periodics.yaml

```diff
@@ -0,0 +1,78 @@
+periodics:
+- name: periodic-kubernetes-containerd-node-throughput-perf-test-ppc64le
+  tags:
+  - "perfDashPrefix: containerd-node-throughput"
+  - "perfDashJobType: throughput"
+  cluster: k8s-ppc64le-cluster
+  decorate: true
+  decoration_config:
+    gcs_configuration:
+      bucket: ppc64le-kubernetes
+      path_strategy: explicit
+    gcs_credentials_secret: gcs-credentials
+  interval: 3h
+  extra_refs:
+  - base_ref: master
+    org: ppc64le-cloud
+    repo: kubetest2-plugins
+    workdir: true
+  - base_ref: master
+    org: kubernetes
+    repo: perf-tests
+  spec:
+    volumes:
+    - name: powercloud-bot-key
+      secret:
+        defaultMode: 256
+        secretName: bot-ssh-secret
+    containers:
+    - image: quay.io/powercloud/all-in-one:0.3
+      command:
+      - /bin/bash
+      volumeMounts:
+      - mountPath: /etc/secret-volume
+        name: powercloud-bot-key
+        readOnly: true
+      envFrom:
+      - secretRef:
+          name: ibm-cloud-credentials
+      args:
+      - -c
+      - |
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+        set -o xtrace
+
+        export PATH=$GOPATH/bin:$PATH
+        export GO111MODULE=on
+
+        go install ./...
+
+        go get sigs.k8s.io/kubetest2@latest
+        go get sigs.k8s.io/kubetest2/...@latest
+
+        TIMESTAMP=$(date +%s)
+        K8S_BUILD_VERSION=$(curl https://storage.googleapis.com/k8s-release-dev/ci/latest.txt)
+        jq --arg key0 'k8s-build-version' --arg value0 $K8S_BUILD_VERSION '. | .[$key0]=$value0' <<<'{}' > $ARTIFACTS/metadata.json
+
+        # kubectl needed for the e2e tests
+        curl -sSL https://dl.k8s.io/ci/$K8S_BUILD_VERSION/bin/linux/`go env GOARCH`/kubectl > /usr/local/bin/kubectl
+        chmod +x /usr/local/bin/kubectl
```

Comment:
The latest nightly `kubectl` installation is needed for the e2e tests.
The all-in-one:0.3 has kubectl with version latest stable release. (https://storage.googleapis.com/kubernetes-release/release/stable.txt)
Please consider if you really need this installation here.

Reply:
Tx @Rajalakshmi-Girish. Whilst I cannot comment on the impact of using a possibly outdated kubectl client, I think having this download is good because - 
- it removes the dependency on the image and having to upgrade it manually just for kubectl version bumps.
- The prow job definitions are not regularly updated which adds a manual dependency on someone having keep updating the prow job with newer images in the definition.
- The download is doing no harm as sort for us because it is being stored in an ephemeral container and we have nothing to lose with an additional step here.
- The existing older jobs also have this step.

Reply:
@mkumatag your opinion on this, please.

Reply:
though all-in-one image got the kubectl binary which is for the generic usecase, for the performance or e2e work, I prefer downloading the matching version.